nifi-registry/nifi-registry-core/nifi-registry-provider-api/src/main/java/org/apache/nifi/registry/hook/EventType.java

```diff
@@ -41,6 +41,14 @@ public enum EventType {
             EventFieldName.VERSION,
             EventFieldName.USER,
             EventFieldName.COMMENT),
+
+    MIGRATE_FLOW_VERSION(
```

Comment:
See later comments about query param approach... If we use the query param approach on the existing end-point then I think this probably goes away